eng/test-determinism.ps1

```diff
@@ -24,9 +22,23 @@ if ($help) {
 
 # List of binary names that should be skipped because they have a known issue that
 # makes them non-deterministic.  
-$script:skipList = @()
-
+$script:skipList = @(
+  # Added to work around https://github.com/dotnet/roslyn/issues/48417
+  "Microsoft.CodeAnalysis.EditorFeatures2.UnitTests.dll",
```

Comment:
This one was already fixed.

Reply:
That change only fixed the field emit, it missed the parameters which have the same problem.